Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -141,14 +141,40 @@ bool FlacLoaderPlugin::parse_header()
     md5_checksum.span().copy_to({ m_md5_checksum, sizeof(m_md5_checksum) });
 
     // Parse other blocks
-    // TODO: For a simple first implementation, all other blocks are skipped as allowed by the FLAC specification.
-    // Especially the SEEKTABLE block may become useful in a more sophisticated version.
+    // TODO: For a simple implementation, all block types except SEEKTABLE and STREAMINFO are skipped as allowed by the FLAC specification.
     [[maybe_unused]] u16 meta_blocks_parsed = 1;
     [[maybe_unused]] u16 total_meta_blocks = meta_blocks_parsed;
     FlacRawMetadataBlock block = streaminfo;
     while (!block.is_last_block) {
         block = next_meta_block(bit_input);
         ++total_meta_blocks;
+
+        if (block.type == FlacMetadataBlockType::SEEKTABLE) {
+            InputMemoryStream steektable_data_memory(block.data.bytes());
+            InputBitStream seektable_data(steektable_data_memory);
+
+            // The number of seek points is implied by the metadata header 'length' field, i.e. equal to length / 18.
+            for (size_t i = 0; i < (block.length / 18); ++i) {
+                // <64> Sample number of first sample in the target frame, or 0xFFFFFFFFFFFFFFFF for a placeholder point.
+                u64 sample_index = seektable_data.read_bits_big_endian(64);
+                if (sample_index == 0xFFFF'FFFF'FFFF'FFFF)
+                    continue;
+
+                // <64> Offset (in bytes) from the first byte of the first frame header to the first byte of the target frame's header.
+                u64 offset = seektable_data.read_bits_big_endian(64);
+
+                // <16> Number of samples in the target frame.
+                u16 sample_count = seektable_data.read_bits_big_endian(16);
+
+                m_seektable.append({
```

Comment:
Since there is an upper bound, I'd suggest calling `m_seektable.ensure_capacity(block.length / 18)` before this loop to avoid reallocations here.